src/Features/VisualBasic/Portable/GenerateConstructorFromMembers/VisualBasicGenerateConstructorFromMembersCodeRefactoringProvider.vb

```diff
@@ -0,0 +1,31 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.GenerateConstructorFromMembers
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.PickMembers
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.GenerateConstructorFromMembers
+    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeRefactoringProviderNames.GenerateConstructorFromMembers), [Shared]>
+    <ExtensionOrder(Before:=PredefinedCodeRefactoringProviderNames.GenerateEqualsAndGetHashCodeFromMembers)>
+    Friend NotInheritable Class VisualBasicGenerateConstructorFromMembersCodeRefactoringProvider
+        Inherits AbstractGenerateConstructorFromMembersCodeRefactoringProvider
+
+        <ImportingConstructor>
+        Public Sub New()
+        End Sub
```

Comment:
rmove.

Reply:
Explicit importing constructors are required for all mef parts. We have analyzer enforcement for this.